skimage/util/tests/test_arraypad.py

```diff
@@ -1,6 +1,8 @@
 from __future__ import division, absolute_import, print_function
+from distutils.version import LooseVersion
 
 import numpy as np
+# note: skimage.util.pad is just numpy.pad
```

Comment:
then, why not calling np.pad directly?

Reply:
(a) this is a change from master backported to make Travis happy, so best to ignore it.
(b) we have historically wrapped pad to harmonize behaviour across numpy versions. Not sure whether it's still needed but not about to fix this in this PR

Reply:
Thanks for the explanation.